agent/consul/stream/convert_test.go

```diff
@@ -135,7 +135,7 @@ func TestConvert_CheckServiceNode(t *testing.T) {
 								LocalBindAddress:     "7.7.7.7",
 								LocalBindPort:        6666,
 								Config: map[string]interface{}{
-									"m": 1,
+									"m": float64(1),
```

Comment:
I wonder if using `require.EqualValues` instead of `require.Equal` would eliminate the need for this. I have started using that when wanting to compare raft index values to prevent having to convert to uint64. EqualValues basically ensures that the two things are equivalent even if not the same type. 

Not that having the float64 calls is a problem. They are fine if we do want to do the type checking too.

Reply:
NIce I'm not sure I've come across `EqualValues` before but I'll consider it in the future!